### PR TITLE
Enable sorting for any column in the abakusleader-view

### PR DESCRIPTION
**Pull request by norbye**
Fixes ABA-186, ABA-188

## Changes
AbakusLeader-view:
* Enable sorting for any column
* Set content width to 80% of screen size instead of 50em for all screen sizes (now 100% width for handheld devices)
    * Table turns to horizontal scroll if the available space is less than 850px
* Set date columns to fit in one line when enough screen-width is available

### Sorting
Default - sorted the way it was returned from the backend
image
Clicked once - ascending
image
Clicked twice - descending
image

### CODE REVIEW

File: frontend/src/containers/AdmissionsContainer/index.tsx [hunk 1]
```diff
@@ -87,19 +95,32 @@ const AdmissionsContainer: React.FC<AdmissionsContainerProps> = ({
...
+                      }
+                      onClick={header.column.getToggleSortingHandler()}
+                    >
+                      {flexRender(
                         header.column.columnDef.header,
                         header.getContext()
                       )}
+                      {{
+                        asc: " 🔼",
+                        desc: " 🔽",
```
Comment (ivarnakken) [inline frontend/src/containers/AdmissionsContainer/index.tsx:120]: We really need to implement an icon library 🙈 Out of scope here though I guess

**Review by ivarnakken — APPROVED** [review]
Good stuff. I love responsive web design 😍

File: frontend/src/containers/AdmissionsContainer/index.tsx (hunk 1 above)
Comment (norbye) [reply frontend/src/containers/AdmissionsContainer/index.tsx]: Kinda forgot that we were already using ionicons for the rest of the project.. updated the feature to use that here as well now:)

Comment (ivarnakken) [reply frontend/src/containers/AdmissionsContainer/index.tsx]: ah nice!

**Pull request merged**